### DISCUSSION TITLE
Getting started with Playground V2.5

### DISCUSSION THREAD

**Discussion by ErmMonzon** [lllyasviel/Fooocus#3133] [Ideas]
Great to see PGv2.5 implemented in Fooocus! Some things to consider to get you started. 

**Guidance Scale** This is personal preference but typically 1-3 is ideal, personally I use 2 mostly. It's very sensitive so only increase by 0.5 increments.
In the advanced tab select "developer debug mode"
![image](https://github.com/lllyasviel/Fooocus/assets/129614926/2bc3dfff-d57b-4551-b52a-6d4cfb80cb0a)

Under sampler you have to change the scheduler to edm_playground_v2.5
Sampler, dpmpp_2m is recommended but you can always experiment. dpmpp_2m_sde_gpu seems to work well, euler and euler_ancestral will give you slightly softer results
![image](https://github.com/lllyasviel/Fooocus/assets/129614926/3de7771f-e615-4eb2-937c-b694dceecf68)

![PGv2 5 CFG](https://github.com/lllyasviel/Fooocus/assets/129614926/6a6f9dfa-b1bc-4bd2-b90b-f4ac0491b4f2)

Close up portraits will give great details but full to half body will likely need some inpainting for faces. Fooocus inpainting model works very well to correct these limitations.

This model has a very different aesthetic, it does have photorealistic capabilities but I find its a very creative model with it's own style and look. 

Let me know if you have any further questions.
Hope this helps!

**Comment by DavidDragonsage** [comment]
What is the appropriate Clip Skip value for Playground 2.5?

**Reply by mashb1t** [reply]
1 or 2, where i personally prefer 2.

**Reply by DavidDragonsage** [reply]
Thank you! 🙂

**Comment by steppres** [comment]
Playground is an unusual model. I've noticed copying a pasting SDXL prompts gives very different results compared to a normal SDXL model. It really, *really* likes to make everything aesthetic (which is the point). It also tends to make everything looks sort of... overly photoshopped? 

I haven't really found a good use for it yet to be honest.
